### DISCUSSION TITLE
Why does the boilerplate for expo use Expo 43?

### DISCUSSION THREAD

**Discussion by jvgeee** [infinitered/ignite#2030] [Q&A] [answered]
Can't run it on a phone using the Expo Go app, Is there any reason you've gone with this instead of V45 like in the bare app?

**Comment by yulolimum** [comment, marked as answer]
Hey @jvgeee . Thanks for your interest in Ignite!

To answer your question, upgrading the Expo template has been on our to-do list for a while now. I also believe there were a couple attempts at doing so, but there were some inconsistencies with using newer Expo CLI versions with Ignite, so no progress was made.

Expo just recently released SDK46 with support for local Expo CLI installs. What does this mean for Ignite? Well, we are currently in the process of revamping the Ignite CLI and template which will ship with Expo 46 and a [unified boilerplate](https://github.com/infinitered/ignite/issues/1983) (meaning you can run it with the Expo CLI or using the vanilla React Native CLI). It is [codenamed Maverick](https://github.com/infinitered/ignite/milestone/19) and you can follow along with the progress. We're close!

**Reply by jvgeee** [reply]
Awesome! Thanks for the update.
